packages/docs-ui/src/components/list-type-picker/picker.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PresetListType } from '@univerjs/core';
+import React from 'react';
+import styles from './index.module.less';
+
+export interface IListTypePickerBaseProps {
+    value?: PresetListType;
+    onChange: (value: PresetListType | undefined) => void;
+}
+
+interface IListTypePickerProps extends IListTypePickerBaseProps {
+    options: { value: PresetListType; img: string }[];
+}
+
+export const ListTypePicker = (props: IListTypePickerProps) => {
+    const { value, onChange, options } = props;
+    return (
+        <div className={styles.docListTypePicker}>
+            {options.map((item) => {
+                return (
+                    <img
+                        key={item.value}
+                        className={styles.docListTypePickerItem + (value === item.value ? ` ${styles.docListTypePickerItemActive}` : '')}
+                        onClick={() => {
+                            onChange(item.value);
+                        }}
+                        src={item.img}
+                    />
+                );
+            })}
+        </div>
+    );
+};
+
+const orderListOptions = [
+    {
+        value: PresetListType.ORDER_LIST,
```

Comment:
These icons should be auto-generated like conditional formatting's icons instead of hard coding.